docs/source/release/v3.7.0/diffraction.rst

```diff
@@ -73,6 +73,21 @@ Improvements in the tomographic reconstruction graphical user interface:
   algorithm :ref:`ImggAggregateWavelengths
   <algm-ImggAggregateWavelengths>`.
 
+
+Powder Diffraction Scripts
+--------------------------
+
+- Pearl legacy powder diffraction has been integrated and can be found
```

Comment:
I'd remove the "legacy" from here. It could sound confusing, as it is being used regularly and it is the preferred alternative.